notebooks/tutorials/intro_for_model_developers.ipynb

```diff
@@ -0,0 +1,1688 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "metadata": {}
+   },
+   "outputs": [],
+   "source": [
+    "test_config = {\n",
+    "    \"validmind.data_validation.ClassImbalance\": {\n",
+    "        \"params\": {\"min_percent_threshold\": 30},\n",
+    "    },\n",
+    "    \"validmind.data_validation.HighPearsonCorrelation\": {\n",
+    "        \"params\": {\"max_threshold\": 0.3},\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "tests_suite = vm.run_documentation_tests(\n",
+    "    inputs={\n",
+    "        \"dataset\": vm_raw_dataset_new,\n",
+    "    },\n",
+    "    config=test_config,\n",
+    "    section=[\"data_preparation\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Log the individual result high correlation test using `vm_raw_dataset` (no data cleanup)\n",
+    "\n",
+    "Here you can use a custom `result_id` to tag the individual result with a unique identifier. This `result_id` can be appended to `test_id` with a `:` separator.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "metadata": {}
+   },
+   "outputs": [],
+   "source": [
+    "result = vm.tests.run_test(\n",
+    "    test_id=\"validmind.data_validation.HighPearsonCorrelation:vm_raw_dataset\",\n",
+    "    params={\"max_threshold\": 0.3},\n",
+    "    inputs={\"dataset\": vm_raw_dataset},\n",
+    ")\n",
+    "result.log()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Log the individual result high correlation test using `vm_raw_dataset_new` (balanced classes and reduced features)\n",
+    "\n",
+    "Repeat the same process as above but with the new dataset, using a new `result_id`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "metadata": {}
+   },
+   "outputs": [],
+   "source": [
+    "result = vm.tests.run_test(\n",
+    "    test_id=\"validmind.data_validation.HighPearsonCorrelation:vm_raw_dataset_new\",\n",
+    "    params={\"max_threshold\": 0.3},\n",
+    "    inputs={\n",
+    "        \"dataset\": vm_raw_dataset_new,\n",
+    "    },\n",
+    ")\n",
+    "result.log()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add individual test results to model documentation\n",
+    "\n",
+    "You can now visit the documentation page for the model you connected to at the beginning of this notebook and add a new content block in the relevant section.\n",
+    "\n",
+    "To do this, go to the documentation page of your model and navigate to the `Data Preparation` -> `Correlations and Interactions` section. Then hover after the \"Pearson Correlation Matrix\" content block to reveal the `+` button as shown in the screenshot below.\n",
+    "\n",
+    "![screenshot showing insert button for test-driven blocks](../images/insert-test-driven-block.png)\n",
```

Comment:
Just minor point here: It would be good if we crop this image specially `Precision Recall Curve` in the image. Here we are inserting text driven block in the `Data Preparation` section but it seems like we are showing `Model performance` tests below. 
<img width="1027" alt="image" src="https://github.com/validmind/developer-framework/assets/30263958/61788349-b6e6-447b-b522-78960a399503">

Reply:
Fixed.